apps/govern/components/Contracts/EditVotes/index.tsx

```diff
@@ -218,7 +219,8 @@ export const EditVotes = ({ allocations, setAllocations, setIsUpdating }: EditVo
       />
 
       <Paragraph type="secondary" className="text-end">
-        New voting weight will take effect at the beginning of the next week.
+        Updated voting weights will take effect at the start of&nbsp;
```

Comment:
`Updated voting weights will take effect at the start of next week (according to Unix time). <Tooltip>Learn more</Tooltip>`

Reply:
Talked to Roman and we agreed to remove the tooltip with the link from this state (where we are editing votes but haven't voted yet) because if no one voted during the week, the link would show an incorrect value, which might be confusing. So we just keep the text `"Updated voting weights will take effect at the start of next week (according to Unix time). "`